custom_components/steam_wishlist/util.py

```diff
@@ -30,15 +31,49 @@ def get_steam_game(game_id: int, game: Dict[str, Any]) -> SteamGame:
         # Price is an integer so $6.00 is 600.
         sale_price = round(int(pricing["price"]) * 0.01, 2)
 
+    reviews_percent = game.get('reviews_percent', 'N/A')
+    review_desc = game.get('review_desc', 'No reviews')
+    rating_info = f"Reviews:&nbsp;&nbsp;{reviews_percent}% ({review_desc})"
+
+    tags = game.get("tags", [])
+    tags_string = ", ".join(tags)
```

Comment:
These aren't used at all.

Reply:
Sorry I forgot to remove that.  That was before I had added `genres` attribute as a string.